services/generic/basic.md

```diff
@@ -1,15 +1,55 @@
-### Basic service
+# Basic Service
 
 A generic service and the most simple way to interact with a device. The actual meaning of "basic" varies from device to device.
 
-#### Service names
+> Please note that basic service is required by Z-Wave specification and serves as a fallback service for devices that are not yet supported by the adapter in order to maintain
+> a rudimentary forward-compatibility. As such this service should not be implemented outside of Z-Wave. Other adapters should use only dedicated services instead.
+
+## Service name
 
 `basic`
 
-#### Interfaces
+## Interfaces
+
+| Type | Interface          | Value type | Description                                  |
+|------|--------------------|------------|----------------------------------------------|
+| in   | cmd.lvl.get_report | null       | Requests report of the set level.            |
```

Comment:
Can we add what message should be returned back? Like:
As a answer remote device returns "evt.lvl.report" message ?
AS it will repreat maybe actually some global rule described will be better.

However with 'cmd.lvl.set; its not so obvous and we should clearly state that as an answer device SHALL respond with "evt.lvl.report"

Reply:
This is going to be part of the general adapter guidlines I will add in future PRs. This is not rewrite of the documentation but refactor and error fixing. Updates will be gradual. Stay put, like, subscribe and push notify button to be informed as soon as new PR are published!